### PR TITLE
Upgrade quickstarts

### PR DESCRIPTION
**Pull request by jschuler**
To latest `@patternfly/quickstarts` package.

### CODE REVIEW

File: .build/src/app/QuickStartDrawerFederated.tsx [hunk 1]
```diff
@@ -1,15 +1,13 @@
+import '@patternfly/react-core/dist/styles/base.css';
```
Comment (jschuler) [inline .build/src/app/QuickStartDrawerFederated.tsx:1]: this will be stripped out in prod

File: .build/webpack.common.js [hunk 2]
```diff
@@ -54,18 +52,6 @@ module.exports = (env, argv) => {
             }
           ]
         },
-        {
```
Comment (jschuler) [inline .build/webpack.common.js:57]: took it out of common and stuck it in the prod config. this way local dev still shows up fine

File: .build/package.json [hunk 3]
```diff
@@ -55,6 +56,7 @@
     "bundle-profile:analyze": "npm run build:bundle-profile && webpack-bundle-analyzer ./stats.json",
     "start": "webpack-cli serve",
     "start:dev": "webpack serve --hot --color --progress --config webpack.dev.js",
+    "start:prod": "serve dist -l 9001 --cors",
```
Comment (jschuler) [inline .build/package.json:59]: to test prod build in the context of the `application-services-ui` container, can first run `npm run build` followed by `npm run start:prod`

**Review by pmuir — APPROVED** [review]

File: .build/webpack.common.js (hunk 2 above)
Comment (pmuir) [reply .build/webpack.common.js:57]: It would be good to try to replicate the prod setup in dev it at all possible i.e. load the pf stylesheet from a cdn.

Comment (jschuler) [reply .build/webpack.common.js:57]: I added a `start:prod` target which will serve up the prod build content so you can load it in `application-services-ui`

**Comment by pmuir** [comment]
@jschuler we have a conflict on this.

**Comment by wtrocki** [comment]
I did run that on localhost for checking what is new. LGTM

**Review by dlabaj — APPROVED** [review]
LGTM

**Comment by dlabaj** [comment]
@pmuir Is there anything else blocking this from merging?

File: .build/webpack.common.js (hunk 2 above)
Comment (pmuir) [reply .build/webpack.common.js:57]: We want to keep the `start:dev` target working for dev - the right way is to just not include the styles in the federated module.

**Comment by pmuir** [comment]
See #244

**Pull request closed without merging**